assets/js/modules/analytics-4/utils/banner-dismissal-expiry.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Calculate expiring GA4 Activation Banner dismissal.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { getTimeInSeconds } from '../../../util/index';
+
+/**
+ * Gets the time in seconds to expire a dismissal of the GA4 Activation Banner.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} referenceDateString Date on which to calculate the expiry for the dismissal.
+ * @return {number} Time in seconds for a dismissal to expire.
+ */
+export function getBannerDismissalExpiryTime( referenceDateString ) {
+	const referenceDate = new Date( referenceDateString );
```

Comment:
Please use the `stringToDate` utility instead of the `Date` constructor here 👍 

More details on why in the docs for that.